cmd/mt-fakemetrics/cmd/parrot.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/grafana/metrictank/schema"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"time"
+)
+
+var (
+	parrotOrgId          int
+	parrotPartitionCount int32
+	partitionMethod      schema.PartitionByMethod
+)
+
+func init() {
+	rootCmd.AddCommand(parrotCmd)
+	parrotCmd.Flags().IntVar(&parrotOrgId, "parrot-org-id", 1, "org id to publish parrot metrics to")
```

Comment:
Does this make more sense as a default than not setting an org ID?

Reply:
Related: I've made the assumption that we're only going to send parrot metrics to a single org.

This disallows partitioning by org.

Reply:
1 is a fine default. single org is fine too I think.